src/components/auth/signup/signup-design.html

```diff
@@ -0,0 +1,59 @@
+
+<div ng-controller="@A.2">
+    <div class="panel-signup" layout="column" layout-paddings>
+        <md-content class="radius-10px md-whiteframe-1dp">
+            <div class="panel-title text-center">
+                <h1 class="no-margin">S3 Portal</h1>
+            </div>
+            
+            <div class="panel-body">
+                <md-subheader class="text-center">Create an account</md-subheader>
+                <form name="signupForm">
+                    <md-input-container md-no-float class="md-icon-left md-block">
+                        <md-icon class="material-icons icon-gray" ng-class="{'md-warn:@A.29'}">email</md-icon>
+                        <md-icon class="material-icons icon-success" ng-if="@A.3">done</md-icon>
+                        <md-progress-circular class="md-accent" md-mode="indeterminate" md-diameter="24" ng-show="@A.4"></md-progress-circular>
+                        
+                        <input name="email" ng-model="@A.5" type="email" placeholder="Email" ng-required="@A.6" ng-email="@A.7" ng-readonly="@A.8" >
+                        
+                        <span class="md-caption md-warn" ng-show="@A.9">Someone already has that username. Try another?</span>
+                        
+                        <div ng-show="@A.10" ng-messages="@A.11">
+                            <div ng-messages-include="@A.12"></div>
+                        </div>
+                    </md-input-container>
+                    
+                    <md-input-container md-no-float class="md-icon-left md-block">
+                        <md-icon class="material-icons icon-gray">lock</md-icon>
+                        
+                        <input name="password" ng-model="@A.13" type="password" placeholder="Password" ng-required="@A.14" ng-readonly="@A.15" >
+                        
+                        <div ng-show="@A.16" ng-messages="@A.17">
+                            <div ng-messages-include="@A.13"></div>
+                        </div>
+                    </md-input-container>
+                    
+                    <md-input-container md-no-float class="md-icon-left md-block">
+                        <md-icon class="material-icons icon-gray">lock</md-icon>
+                        
+                        <input name="password_confirm" ng-model="@A.18" type="password" placeholder="Retype Password" ng-required="@A.19" ng-equals="@A.20" ng-readonly="@A.21" >
```

Comment:
Input name should be assigned by F2E.
ng-required should be always be true here.